pkg/retrieval/retrieval.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:generate sh -c "protoc -I . -I \"$(go list -f '{{ .Dir }}' -m github.com/gogo/protobuf)/protobuf\" --gogofaster_out=. retrieval.proto"
```

Comment:
This go:generate line is not valid currently as the proto file is no longer in this directory. The line should be moved into a doc.go file in pb directory.